web_src/js/features/repo-settings.js

```diff
@@ -65,3 +65,52 @@ export function initRepoSettingBranches() {
     });
   }
 }
+
+export function initRepoSettingsSSHAuthorization() {
+  const generateButton = document.querySelector('#generate-ssh-key');
+  if (!generateButton) {
+    return;
+  }
+  const deleteSSHButton = document.querySelector('#delete-ssh-key');
```

Comment:
I didn't find '#generate-ssh-key' and '#delete-ssh-key' in the latest code, am I missing something?

Reply:
Left-overs, no JS is needed anymore.